lib/rules/decorator-position.js

```diff
@@ -265,6 +302,25 @@ function configuredDecoratorsInOptions(options) {
   return allConfigs.map((config) => config[0]);
 }
 
+function lineLength(userOptions, filePath) {
+  if (!prettier) {
+    prettier = require('prettier');
+  }
+
+  const eslintPrettierOptions = cli.getConfigForFile(filePath).rules['prettier/prettier'][1] || {};
+  const usePrettierrc = !eslintPrettierOptions || eslintPrettierOptions.usePrettierrc !== false;
+  const prettierRcOptions = usePrettierrc
+    ? prettier.resolveConfig.sync(filePath, {
+        editorconfig: true,
+      })
+    : {};
+
+  const prettierOptions = Object.assign({}, prettierRcOptions, eslintPrettierOptions, {
+    filePath,
+  });
+  return { ...userOptions, ...prettierOptions.printWidth };
```

Comment:
this might be a bug?
```ts
   return { 
     ...userOptions, 
     printWidth: prettierOptions.printWidth 
   };
```
might be what's causing the tests to fail